packages/connect/src/ScProvider.ts

```diff
@@ -57,6 +57,7 @@ export class ScProvider implements ProviderInterface {
    * @param displayName - a display name that will be used from the dev-tools of the Extension (if installed)
    * @param knownChain - the name of a supported chain ({@link SupportedChains})
    * @param parachainSpec - optional param of the parachain chainSpecs to connect to
+   * @param autoConnect - optional param that defaults to True,
```

Comment:
```suggestion
   * @param autoConnect - whether the ScProvider should eagerly connect while its being instantiated. Defaults to `true`
```

Also, please add the same comment to the other overload.